crates/vhost-device-gpio/src/gpio.rs

```diff
@@ -9,11 +9,12 @@ use log::error;
 use std::sync::{Arc, Mutex, RwLock};
 use std::time::Duration;
 
+use crate::virtio_gpio::*;
```

Comment:
This should go after this section:
https://rust-lang.github.io/rustfmt/?version=v1.6.0&search=#StdExternalCrate\%3A